_rules/button-accessible-name-97a4e1.md

```diff
@@ -168,59 +168,32 @@ Off screen element without an [accessible name][].
 </html>
 ```
 
-#### Failed Example 4
-
-Button has an [accessible name][] that is only [whitespace][].
-
-```html
-<button></button>
-```
-
 ### Inapplicable
 
 #### Inapplicable Example 1
 
-Image buttons are tested in a different rule.
+Image buttons are tested in a separate rule, because also have to be tested for [success criterion 1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content).
```

Comment:
Surely image buttons are still required to have an accessible name.

Reply:
I don't think this should hold up this PR. A new on can be created if needed following completion of discussions at #1049 .